### PR TITLE
Add Ubuntu default config path

### PR DESCRIPTION
**Pull request by neomilium**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -1,10 +1,18 @@
 class firefox::params {
   $owner = 'root'
 
-  case $::osfamily {
+  case $::os['family'] {
     'debian': {
-      $config = '/usr/share/firefox-esr/browser/defaults/preferences/00-puppet-preferences.js'
-      $group = 'root'
+      case $::os['name'] {
```
Comment (smortex) [inline manifests/params.pp]: `$facts['os']['name']`  (This would cause build failure)

File: manifests/params.pp [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+        'Ubuntu': {
```
Comment (smortex) [inline manifests/params.pp]: `'ubuntu'` (string comparison is case insensitive)

**Comment by smortex** [comment]
`pamas.pp` has been [considered a smell for a while](https://www.devco.net/archives/2013/12/09/the-problem-with-params-pp.php).  Maybe a good opportunity to embrace [Hiera 5](https://puppet.com/resources/webinar/introduction-hiera-5/thank-you) (or Hiera 4 if Puppet 4 compatibility is required)

**Comment by smortex** [comment]
Just opened #4 that adds support for Hiera 5

**Comment by neomilium** [comment]
Branch rebased on master.
Changed applied.

Thanks for review.

**Pull request closed without merging**